dotnet/src/SemanticKernel.Abstractions/AI/ChatCompletion/ChatMessageBase.cs

```diff
@@ -17,14 +17,20 @@ public abstract class ChatMessageBase
     /// </summary>
     public string Content { get; set; }
 
+    /// <summary>
+    /// Content of the message
+    /// </summary>
+    public string? Name { get; set; }
+
     /// <summary>
     /// Creates a new instance of the <see cref="ChatMessageBase"/> class
     /// </summary>
     /// <param name="role">Role of the author of the message</param>
     /// <param name="content">Content of the message</param>
-    protected ChatMessageBase(AuthorRole role, string content)
+    protected ChatMessageBase(AuthorRole role, string content, string? name = null)
```

Comment:
If we don't want to change the constructor, can alternatively have caller set the Name property once the object has been created.

Reply:
Or add a separate constructor with the extra parameter